src/hardware/plugin/distance_sensors_plugin.c

```diff
@@ -27,13 +27,18 @@ static const detect_enemy_keys_t detect_right_keys = {
   .enemy_detected = key_right_sensor_enemy_detected
 };
 
+static volatile tiny_adc_counts_t c;
 static void read_sensor(
   distance_sensors_plugin_t* self,
   tiny_adc_channel_t channel,
   tiny_key_value_store_key_t key) {
   tiny_adc_counts_t counts = tiny_adc_group_read(self->adc_group, channel);
   distance_in_cm_t distance = gp2y0a21yk0f_counts_to_distance(counts);
   tiny_key_value_store_write(self->key_value_store, key, &distance);
+  if(channel == left_channel) {
+    tiny_key_value_store_write(self->key_value_store, key_knob_counts, &counts);
+    c = counts;
+  }
```

Comment:
I feel like this might not belong here

Reply:
it doesn't, but I wanted to leave it in until I can find a channel that works properly

Reply:
this way we can swap the knob and the distance sensor as needed until it's fully integrated